Signal/src/view controllers/CallViewController.swift

```diff
@@ -519,7 +519,6 @@ class CallViewController: UIViewController, CallDelegate {
 
     func updateCallUI(callState: CallState) {
         let textForState = localizedTextForCallState(callState)
-        Logger.info("\(TAG) new call status: \(callState) aka \"\(textForState)\"")
```

Comment:
Sure, that's a reasonable optimization. But IMO the value of the logging is to surface the changes in the state, not to document calls to updateCallUI(), so my inclination is to move it to the state setter.